src/data/data.json

```diff
@@ -1,7 +1,7 @@
 {
     "classifications": [
         { 
-            "index": 1,
+            "index": 0,
             "name": "Town",
             "img": "../assets/images/classification/Town.webp",
```

Comment:
you can just keep the file name and use the path when you are using it in the img src